internal/osbuild2/ostree_fillvar_stage.go

```diff
@@ -0,0 +1,24 @@
+package osbuild2
+
+// Options for the org.osbuild.ostree.fillvar stage.
+type OSTreeFillvarStageOptions struct {
+
+	Deployment Deployment `json:"deployment"`
+}
+
+type Deployment struct {
```

Comment:
Same as with `ConfigOptions`, should probably rename this to `OSTreeDeployment`.

Reply:
fixed!